Dockerfile

```diff
@@ -19,14 +19,18 @@ RUN . /build.env \
  && git checkout ${VPP_COMMIT} \
  && UNATTENDED=y make install-dep bootstrap pkg-deb \
  && cd build-root \
- && tar -zcf vpp-debs.tar.gz ./*.deb
+ && bash -c "tar -czvf vpp-debs.tar.gz {vpp,vpp-plugins,vpp-lib}_$(git describe --tags | sed -e s/-/~/2 -e s/^v//)_amd64.deb" \
+ && mv ./install-vpp-native/vpp/lib64/ /opt/ldpreload/ \
+ && tar -czvf vpp-ldpreload.tar.gz /opt/ldpreload/
 
 FROM ubuntu:16.04
 
 COPY --from=builder /vpp/build-root/vpp-debs.tar.gz /tmp/vpp-debs.tar.gz
+COPY --from=builder /vpp/build-root/vpp-ldpreload.tar.gz /tmp/vpp-ldpreload.tar.gz
```

Comment:
Could you copy the libraries directly to their destinations instead? This is relevant if the resulting image size is relevant.

Reply:
done